src/rollup/RollupCreator.sol

```diff
@@ -114,6 +121,19 @@ contract RollupCreator is Ownable {
             })
         );
 
+        sequencerInbox.setIsBatchPoster(_batchPoster, true);
+
+        // Call setValidator on the newly created rollup contract
+        bool[] memory _vals = new bool[](_validators.length);
+        for (uint256 i = 0; i < _validators.length; i++) {
+            _vals[i] = true;
+        }
+        IRollupAdmin(address(rollup)).setValidator(_validators, _vals);
+
+        IRollupAdmin(address(rollup)).setOwner(actualOwner);
+
+        proxyAdmin.transferOwnership(actualOwner);
```

Comment:
you should move it back to the original place before config.owner is changed and use config.owner

Reply:
Done

Reply:
did you forgot to push?

Reply:
LOL! Yup! WTF sorry!